batched/sparse/src/KokkosBatched_Krylov_Handle.hpp

```diff
@@ -80,10 +80,10 @@ class KrylovHandle {
   bool host_synchronised;
 
  public:
-  KrylovHandle(int _batched_size, int _N_team, int _max_iteration = 200, bool _monitor_residual = false)
```

Comment:
Here you could just rename the constructor parameter instead of the struct's attribute, it would require only once change: `_N_team` to `N_team_`